lib/transformations/extractTextPlugin/extractTextPlugin.js

```diff
@@ -1,32 +1,39 @@
+// @flow
+import type { Ijscodeshit, IPath, INewExpression, IObjectExpression, IMemberExpression, ILiteral, ICallExpression } from '../../types'; // eslint-disable-line node/no-unsupported-features
 const utils = require('../utils');
 
-function findInvocation(j, node, pluginName) {
-	return j(node)
+function findInvocation(j: Ijscodeshit, node: IPath<IMemberExpression>, pluginName: string): boolean {
```

Comment:
I think this is changing the implementation of the function completely. `j(node)` isn't boolean and will return a node AFAIK.

Reply:
Oh my bad! You're right!